cmd/kops/delete_cluster.go

```diff
@@ -180,7 +176,7 @@ func RunDeleteCluster(f *util.Factory, out io.Writer, options *DeleteClusterOpti
 
 			fmt.Fprintf(out, "\n")
 
-			err = d.DeleteResources(clusterResources)
+			err = resources.DeleteResources(cloud, clusterResources)
```

Comment:
Can we put `DeleteResources` into `resource/utils` to be consistent with `ListResources`? Or does that create a circular dependency?

Reply:
We definitely can :-)  Did that as a separate additional commit in #3720 to (hopefully) make it easier to review.

This does suggest that maybe this should be `pkg/resource/operations` instead of `pkg/resources/utils` - maybe?